src/NugetReferenceResolver/NugetReferenceResolver.csproj

```diff
@@ -9,13 +9,9 @@
     <OutputType>library</OutputType>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <OutputType>library</OutputType>
-    <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
-  </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(AspNetCoreVersion)" />
```

Comment:
Can you add a different version variable for this one? We don't produce this package and it may not be the same as AspNetCoreVersion. For example, its [latest version](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.DotNet.PlatformAbstractions) is 2.0.0-\*, not 1.2.0-\*